portugues/colabora/fork/comdiferencas/diff/pull/desisto/desistir.md

```diff
@@ -1,2 +1,4 @@
 Quando você desiste você descobre que ainda precisa usar o programa que tem o bug e  
-que não tem outra opção senão conseguir arrumar o bug.
\ No newline at end of file
+que não tem outra opção senão conseguir arrumar o bug.
+
+Voltar para o [inicio](../../../../../../)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.